architecture/additional_concepts/service_catalog.adoc

```diff
@@ -0,0 +1,321 @@
+catalog.
+
+[NOTE]
+====
+Because the service catalog is currently Technology Preview, the following
+service brokers are also currently Technology Preview.
+====
+
+[[service-catalog-template-service-broker]]
+=== Template Service Broker
+
+The _template service broker_ gives the service catalog visibility into the
+xref:../../dev_guide/templates.adoc#using-the-instantapp-templates[default Instant App and Quickstart templates] that have shipped with {product-title}
+since its initial release. By default, the template service broker shows the
+objects that are globally available from the *openshift* project. It can also be
+configured to watch any other project that a cluster administrator chooses.
+
+ifndef::openshift-online,openshift-dedicated[]
+See _Deploying the Template Service Broker_ (TODO link) for more details.
+endif::[]
+
+[[service-catalog-asb]]
+=== Ansible Service Broker
+
+The Ansible service broker is an implementation of the OSB API that manages
+applications defined by xref:service-catalog-apb[_Ansible playbook bundles (APBs)_]. APBs provide a new method for defining and distributing container
+applications in {product-title}, consisting of a bundle of Ansible playbooks
+built into a container image with an Ansible runtime. APBs leverage Ansible to
+create a standard path for transitioning from easy to complex deployments.
+
+The design of the Ansible service broker follows this basic workflow:
+
+. A user requests list of available applications from the service catalog using
+the {product-title} web console.
+. The service catalog requests the Ansible service broker for available
+applications.
+. The Ansible service broker communicates with a defined container registry to
+learn which APBs are available.
+. The user issues a request to provision a specific APB.
+. The provision request makes its way to the Ansible service broker, which
+fulfills the user's request by invoking the provision method on the APB.
+
+image::asb-arch.png["Ansible Service Broker Architecture"]
+
+ifndef::openshift-online,openshift-dedicated[]
+See _Deploying the Ansible Service Broker_ (TODO link) for more details.
+endif::[]
+
+[[service-catalog-apb]]
+==== Ansible Playbook Bundles
+
+An Ansible playbook bundle (APB) is a lightweight application definition
+(meta-container). They are used to define and deploy complex groups of
+applications, deployment configurations, deployments, and services to an
+{product-title} cluster running the Ansible service broker.
+
+APBs offer more power and simple configuration by leveraging the power of
+Ansible. APBs have the following features:
+
+- Metadata contains list of required and optional parameters for use during deployment.
+- Leverages existing investment in Ansible roles and playbooks.
+- Actions under a directory with named playbooks and metadata defined in *_apb.yml_*.
+- Developer tooling to drive a guided approach.
+- Easily modified or extended.
+
+[[service-catalog-apb-directory]]
```

Comment:
We've gone a long way from architecture additional concepts at this point.  Surely a separate document?